CFC_DataCollector/recommender/recommendation_pipeline.py

```diff
@@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
         return []
 
     def get_selected_user_utility_model(self, user_id):
-        return []
+        return UserUtilityModel.find_from_db(user_id)
 
     def recommend_trips(self, trip_id, utility_model):
         return []
```

Comment:
Actually, I am not sure that makes sense. I am pretty sure that Robert said that the answer was to multiply the probabilities together, but I don't think that we want to multiply the probabilities of the parameters together.